generators/client/types.d.ts

```diff
@@ -30,6 +31,9 @@ export type Application<E extends Entity = Entity> = Command['Application'] &
     clientWebappDir?: string;
     clientThemeNone?: boolean;
     clientThemeAny?: boolean;
+    clientRootDir: string;
+    clientSrcDir: string;
+    clientTestDir?: string;
```

Comment:
Should be removed from BaseApplication definitions

Reply:
They are I think